governance/tech-lead-proposal-process.md

```diff
@@ -0,0 +1,107 @@
+<!-- cSpell:ignore chairpersonship markdownlint -->
+# Security TAG Technical Lead Proposal Process
+
+Security Technical Leads operate as a team alongside the Co-Chairs. The Co-Chair
+team seeks enhance the leadership capabilities of the existing co-chairs across
+the key qualities of industry experience, hands-on cloud-native and security
+experience, as well as administrative experience needed to assist in the
+execution of the various projects and efforts that make the Security TAG so
+successful.
+
+## Caveats of Technical Leads
+
+* The position of a technical lead does not guarantee future co-chairpersonship
+* The position of a technical lead is not a paid position (co-chairs are also
+  not paid positions)
+* Technical leads [have responsibilities](roles.md) to the Security TAG, the
+  leadership team, and the TOC
+* Nominations by the community (with co-chair endorsement) or co-chairs still
+  [requires TOC
+  approval](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#elections)
+
+## Process of nominations
+
+Depending on the needs of the community, as determined by the Security TAG
+leadership team, technical lead nominations may come in the form of "Nominees by
+community", "Nominees by co-chairs", or both.
+
+The Security TAG does not have a current limit on the amount of technical leads.
+
+Final technical lead nominations will adhere to the [CNCF's TOC described
+process](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#elections).
+
+### Nominees by community
+
+Community nominees should adhere to the technical lead nomination requirements
+described below.
+
+The co-chairs' reserve the right to endorse a nominee by the community based on:
+
+* Gender diversity
+* Company diversity
+* Geo diversity
+* Amount of nominations from different companies, people, regions for a single
+  individual
+* Existing body of work
+* Justification provided during nomination
+
+#### Community timeline
+
+Community nominations will to run for at least 1 month with a maximum up to 2
+months. Upon close of nominations a decision will be made by the co-chairs
+within the next month in order conform to the TOC process.
+
+### Nominees by co-chairs
+
+Nominees by co-chairs adhere to the technical lead requirements described below.
+Co-chairs are responsible for documenting statistic information of all nominee
+types for endorsement and presentation to the TOC.
+
+#### Endorsement of community nominees
```

Comment:
I think this goes above the `### Nominees by co-chairs` since it is still under `Nominees by community` process